packages/rrweb-snapshot/src/snapshot.ts

```diff
@@ -266,20 +267,34 @@ export function transformAttribute(
     return absoluteToDoc(doc, value);
   } else if (
     maskAllText &&
-    (['placeholder', 'title', 'aria-label'].indexOf(name) > -1 ||
-      (tagName === 'input' &&
-        name === 'value' &&
-        element.getAttribute('type') &&
-        ['submit', 'button'].indexOf(
-          element.getAttribute('type')!.toLowerCase(),
-        ) > -1))
+    _shouldMaskAttribute(element, name, tagName, { unmaskTextSelector })
   ) {
     return maskTextFn ? maskTextFn(value) : defaultMaskFn(value);
   }
 
   return value;
 }
 
+function _shouldMaskAttribute(
+  element: HTMLElement,
+  name: string,
```

Comment:
Maybe `attribute` or `attributeName` to distinguish it a bit from `tagName`:


```suggestion
  attribute: string,
```